### PR TITLE
Various minor fixes for the output snippet and UI

### PR DESCRIPTION
**Pull request by stevesan**

### CODE REVIEW

File: packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts [hunk 1]
```diff
@@ -52,17 +54,28 @@ export class ExportPanel extends ConnectedLitElement {
...
-    if (editedConfig.environmentImage) {
-      editedConfig.environmentImage = `environment-image.hdr`;
+
+    // If the last loaded URL is not an object URL, echo it here for convenience.
+    if(this.gltfUrl && !isObjectUrl(this.gltfUrl)) {
+      editedConfig.src = this.gltfUrl;
+    }
+    else {
+      // Uploaded GLB
+      editedConfig.src = `Change this to your GLB URL`;
```
Comment (elalish) [inline packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts:70]: 👍

File: packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts [hunk 2]
```diff
@@ -52,17 +54,28 @@ export class ExportPanel extends ConnectedLitElement {
...
+    }
+
+    if (editedConfig.environmentImage &&
+      isObjectUrl(editedConfig.environmentImage)) {
+      // Uploaded env image
+      editedConfig.environmentImage = `Change this to your HDR URL`;
```
Comment (elalish) [inline packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts:76]: Should we do the same thing here as above, to retain the url if it's not an Object URL?

File: packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts [hunk 3]
```diff
@@ -123,13 +136,16 @@ export class ModelViewerSnippet extends LitElement {
...
       try {
-        dispatchGltfUrl(undefined);
-        // Because of update-batching, we need to sleep first to force reload.
-        await new Promise(resolve => {
-          setTimeout(resolve, 0);
-        });
-
-        dispatchGltfUrl(config.src);
+        // If we can't fetch the snippet's src, don't even bother using it.
+        // But still dispatch the config, hotspots, etc.
```
Comment (elalish) [inline packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts:140]: 👍

File: packages/space-opera/src/components/shared/snippet_viewer/snippet_viewer.ts [hunk 4]
```diff
@@ -59,7 +59,7 @@ export class SnippetViewer extends LitElement {
   }
 
   protected updated() {
-    this.snippet.textContent = this.formattedStyle + this.formattedHtml;
+    this.snippet.textContent = this.formattedHtml + '\n' + this.formattedStyle;
```
Comment (elalish) [inline packages/space-opera/src/components/shared/snippet_viewer/snippet_viewer.ts:62]: 👍

File: packages/space-opera/src/components/shared/snippet_viewer/snippet_viewer.ts [hunk 5]
```diff
@@ -71,8 +71,13 @@ ${css_beautify(this.renderedStyle)}
...
 
   get formattedHtml() {
-    const html = this.shadowTag.innerHTML.replace(
-        /<!---->/g, '');  // Removes LitElement render artifacts
+    // Removes LitElement render artifacts
+    let html = this.shadowTag.innerHTML.replace(
+        /<!---->/g, '');
+    // Remove the ar-status runtime-added tag
+    html = html.replace(/ar-status="[\w- ]+" */, '');
+    // Remove redundant ="" for boolean attribs
```
Comment (elalish) [inline packages/space-opera/src/components/shared/snippet_viewer/snippet_viewer.ts:79]: 👍

File: packages/space-opera/src/redux/initial_environment_images.ts [hunk 6]
```diff
@@ -22,11 +22,11 @@ import {EnvironmentImage} from './lighting_state.js';
  */
 export const INITIAL_ENVIRONMENT_IMAGES: EnvironmentImage[] = [
   {
-    uri: '../shared-assets/environments/spruit_sunrise_1k_HDR.hdr',
+    uri: './node_modules/@google/model-viewer-shared-assets/environments/spruit_sunrise_1k_HDR.hdr',
     name: 'Spruit Sunrise'
   },
   {
-    uri: '../shared-assets/environments/aircraft_workshop_01_1k.hdr',
+    uri: './node_modules/@google/model-viewer-shared-assets/environments/aircraft_workshop_01_1k.hdr',
```
Comment (elalish) [inline packages/space-opera/src/redux/initial_environment_images.ts:29]: Why this change, out of curiosity? Seems like it works both ways?

File: packages/space-opera/src/redux/space_opera_base.ts [hunk 7]
```diff
@@ -225,3 +225,20 @@ export const dispatchCurrentCameraState = registerStateMutator(
       if (!currentCamera) return;
       state.currentCamera = {...currentCamera};
     });
+
+/**
+ * Convenience function for components that import GLBs.
+ * We consider "staging config" to be properties that are applicable to any
+ * model, and thus are sensible to preserve when a new model is loaded.
```
Comment (elalish) [inline packages/space-opera/src/redux/space_opera_base.ts:232]: 💯

File: packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts (hunk 2 above)
Comment (stevesan) [reply packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts:76]: We already do retain it, so this code does that.

File: packages/space-opera/src/redux/initial_environment_images.ts (hunk 6 above)
Comment (stevesan) [reply packages/space-opera/src/redux/initial_environment_images.ts:29]: It does, but this will work even if you serve from packages/space-opera directly, for the "npm run dev" case
